app/imports/client/react/forms/components/DepartmentsSelectInput.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { mapEntitiesToOptions } from 'plio-util';
+
+import { Query } from '../../../graphql';
+import { swal } from '../../../util';
+
+import ApolloSelectInputField from './ApolloSelectInputField';
+
+const DepartmentsSelectInput = ({
+  organizationId,
+  type,
+  onNewOptionClick,
+  ...props
+}) => (
+  <ApolloSelectInputField
+    {...props}
+    onNewOptionClick={onNewOptionClick}
+    type={type || onNewOptionClick ? 'creatable' : undefined}
```

Comment:
It is better to replace and move to destructuring of props of the component
```suggestion
const DepartmentsSelectInput = ({
    organizationId,
    onNewOptionClick,
    type = onNewOptionClick && 'creatable',
    ...props
}) => (
```

Reply:
It is even better to leave this up to user so he should supply `creatable` prop when using `onNewOptionClick`. You can just default this prop to 'creatable'.